src/schemas/transaction.yaml

```diff
@@ -1,3 +1,86 @@
+Transaction7702Signed:
+  type: object
+  title: EIP-7702 transaction.
+  required:
+    - type
+    - nonce
+    - to
+    - gas
+    - value
+    - input
+    - maxPriorityFeePerGas
+    - maxFeePerGas
+    - maxFeePerBlobGas
+    - accessList
+    - blobVersionedHashes
+    - chainId
+    - authorization list
+  properties:
+    type:
+      title: type
+      type: string
+      pattern: ^0x2$
+    nonce:
+      title: nonce
+      $ref: '#/components/schemas/uint'
+    to:
+      title: to address
+      oneOf:
+        - title: Contract Creation (null)
+          type: 'null'
+        - title: Address
+          $ref: '#/components/schemas/address'
+    gas:
+      title: gas limit
+      $ref: '#/components/schemas/uint'
+    value:
+      title: value
+      $ref: '#/components/schemas/uint'
+    input:
+      title: input data
+      $ref: '#/components/schemas/bytes'
+    maxPriorityFeePerGas:
+      title: max priority fee per gas
+      description: Maximum fee per gas the sender is willing to pay to miners in wei
+      $ref: '#/components/schemas/uint'
+    maxFeePerGas:
+      title: max fee per gas
+      description: The maximum total fee per gas the sender is willing to pay (includes the network / base fee and miner / priority fee) in wei
+      $ref: '#/components/schemas/uint'
+    maxFeePerBlobGas:
+      title: max fee per blob gas
+      description: The maximum total fee per gas the sender is willing to pay for blob gas in wei
```

Comment:
Nit:
It isn't consistent across the file. I suggest using either periods or no periods for all descriptions.

Reply:
Didn't understood what you meant  ?

Reply:
Some descriptions have periods while others do not.

Reply:
Like not able to understand what's meant by period here I just copied tx4844 same there as well is that wrong as well can you show me an example change by suggesting one

Reply:
removed periods